cookbook-connector/src/test/resources/AutomationSpringBeans.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file is required, as its being used by the TestBuilder to populate the test data.